app/styles/_membership.less

```diff
@@ -31,7 +31,7 @@
     .col-name {
       min-width: 165px;
       padding: 0 5px 10px 0;
-      word-wrap: break-word;
+      word-break: break-all;
```

Comment:
With `word-wrap: break-word` you get this:
![screen shot 2016-11-01 at 1 16 43 pm](https://cloud.githubusercontent.com/assets/280512/19899185/809e2300-a035-11e6-85df-33b667433996.png)